code/modules/antagonists/malf_ai/datum_malf_ai.dm

```diff
@@ -52,7 +52,7 @@
 	var/objective_limit = CONFIG_GET(number/traitor_objectives_amount)
 	var/objective_count = length(objectives)
 
-	for(var/i in objective_count to objective_limit)
+	for(var/i = objective_count, i < objective_limit, ++i)
```

Comment:
huh?

Reply:
oh i decided to read the pr

Reply:
just do `for (var/i in objective_count to objective_limit - 1)` tbqh

Reply:
Feexed.